sympy/solvers/inequalities.py

```diff
@@ -263,8 +264,11 @@ def reduce_rational_inequalities(exprs, gen, relational=True):
 
     if eqs:
         solution &= solve_rational_inequalities(eqs)
+        exclude = solve_rational_inequalities([[((d, Poly(S.One, gen)), '==')
+            for i in eqs for ((n, d), _) in i if d.has(gen)]])
+        solution -= exclude
 
-    if not exact:
+    if not exact and solution:
```

Comment:
A test was added for this. If the solution is `S.EmptySet` it cannot be evalf'ed. The alternative is to make `S.EmptySet.evalf()` return something but I'm not sure what. `S.Reals.evalf()` returns `Interval(-oo, oo)`; S.EmptySet and S.UniversalSet both raise errors.